dvc/command/add.py

```diff
@@ -74,6 +77,24 @@ def add_parser(subparsers, parent_parser):
         help="Specify name of the DVC-file this command will generate.",
         metavar="<filename>",
     )
+    parser.add_argument(
+        "-o",
+        "--out",
+        help="Destination path to put files to.",
+        metavar="<filename>",
```

Comment:
Consistent with `import`:
```suggestion
        help="Destination path to put files to",
        metavar="<path>",
```